### ISSUE TITLE
single item select doesn't hide dropdown after choose

### ISSUE THREAD

**Issue by kot-pilot** [nathanboktae/knockout-choose#5]
single item select doesn't hide dropdown after choose

**Comment by nathanboktae** [comment]
It works fine in [the demo site](http://nathanboktae.github.io/knockout-choose/) and I [have a test for it](https://github.com/nathanboktae/knockout-choose/blob/master/tests/tests.js#L171-L187) also.

Do you have more information you can provide?

**Comment by kot-pilot** [comment]
open your example in ie 11

**Comment by nathanboktae** [comment]
Yeah the IE part is pretty important. Looks like it's flickering likely due to an ordering issue with click, blur, and/or focus events.
